<launch>
    <arg name="map_size_x" value="24"/>
    <arg name="map_size_y" value="24"/>
    <arg name="map_size_z" value="7"/>
    <arg name="odom_topic" value="visual_slam/odom" />
    <!-- <arg name="drone_id" value="0"/>
    <arg name="init_x" value="0"/>
    <arg name="init_y" value="0"/>
    <arg name="init_z" value="0"/> -->
    
    <include file="$(find ego_planner)/launch/include/run_in_sim.xml">
        <arg name="drone_id" value="$(arg drone_id)"/>
        <arg name="init_x" value="$(arg init_x)"/>
        <arg name="init_y" value="$(arg init_y)"/>
        <arg name="init_z" value="$(arg init_z)"/>
        <arg name="map_size_x" value="$(arg map_size_x)"/>
        <arg name="map_size_y" value="$(arg map_size_y)"/>
        <arg name="map_size_z" value="$(arg map_size_z)"/>
        <arg name="odom_topic" value="$(arg odom_topic)"/>
        <!-- <arg name="point_num" value="5"/>
        <arg name="target0_x" value="0"/>
        <arg name="target0_y" value="8"/>
        <arg name="target0_z" value="1"/>
        <arg name="target1_x" value="8"/>
        <arg name="target1_y" value="0"/>
        <arg name="target1_z" value="1"/>
        <arg name="target2_x" value="0"/>
        <arg name="target2_y" value="-8"/>
        <arg name="target2_z" value="1"/>
        <arg name="target3_x" value="-8"/>
        <arg name="target3_y" value="0"/>
        <arg name="target3_z" value="1"/>
        <arg name="target4_x" value="0"/>
        <arg name="target4_y" value="0"/>
        <arg name="target4_z" value="1"/> -->
    </include>
</launch>
    